packages/edit-navigation/src/store/reducer.js

```diff
@@ -98,8 +98,23 @@ export function selectedMenuId( state = null, action ) {
 	return state;
 }
 
+/**
+ * Reducer tracking whether the inserter is open.
+ *
+ * @param {boolean|Object} state
+ * @param {Object}         action
+ */
+function blockInserterPanel( state = false, action ) {
```

Comment:
Is it cleaner to deconstruct with object? 
```suggestion
function blockInserterPanel( state = false, { type, value } ) {
```

Reply:
For now, I'm following the pattern established elsewhere in the file.